tests/examples/__snapshots__/example26.snap.svg

```diff
@@ -0,0 +1,157 @@
+<svg width="640" height="640" viewBox="0 0 640 640" xmlns="http://www.w3.org/2000/svg">
```

Comment:
the behavior of this snapshot is a regression i believe, calculate-elbow shouldn't generate a path like this